db/migrate/20230502183501_create_comments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :author_id
+      t.references :post, null: false, foreign_key: true
+      t.text :text
+
+      t.timestamps
+    end
+    add_index :comments, :author_id
```

Comment:
- [ ] Kindly do the same here. For more documentation, ou can visit this link about [Creating associations using references](https://edgeguides.rubyonrails.org/active_record_migrations.html#creating-associations-using-references)